src/desktop/main.js

```diff
@@ -216,7 +216,17 @@ function createWindow() {
         if (url.indexOf(targetURL) !== 0) {
             e.preventDefault();
 
-            const externalWhitelist = ['iota.org', 'docs.iota.works', 'trinity.iota.org', 'docs.bugsnag.com'];
+            const termsAndConditionsLinks = ['iota.org', 'trinity.iota.org', 'mailto:contact@iota.org'];
+            const privacyPolicyLinks = [
+                'corp.sogou.com',
+                'cryptocompare.com',
+                'docs.bugsnag.com',
+                'help.github.com',
+                'policies.google.com',
+                'protect-eu.mimecast.com',
```

Comment:
Just noticed one more, sorry. I think `mailto:privacy@iota.org` also needs to be whitelisted